src/components/AppNavigation/CalendarList/CalendarListItem.vue

```diff
@@ -27,22 +27,16 @@
 		:class="{deleted: !!deleteTimeout, disabled: !calendar.enabled, 'open-sharing': shareMenuOpen}"
 		@click.prevent.stop="toggleEnabled">
 		<template slot="icon">
-			<Actions>
-				<ActionButton @click.prevent.stop="toggleEnabled">
-					<template #icon>
-						<CheckboxBlankCircle
-							v-if="calendar.enabled"
-							:title="$t('calendar', 'Disable calendar')"
-							:size="20"
-							:fill-color="calendar.color" />
-						<CheckboxBlankCircleOutline
-							v-else
-							:title="$t('calendar', 'Enable calendar')"
-							:size="20"
-							:fill-color="calendar.color" />
-					</template>
-				</ActionButton>
-			</Actions>
+			<CheckboxBlankCircle
+				v-if="calendar.enabled"
+				:title="$t('calendar', `Disable ${calendar.displayName.toLowerCase()} calendar`)"
```

Comment:
This won't work with the translations. You have to use it like this (untested code):
```suggestion
				:title="$t('calendar', 'Disable calendar {calendar}', {calendar: calendar.displayName})"
```

Reply:
See
https://github.com/nextcloud/calendar/blob/0467c6e5571a9346b33756da43fea050b30c5f59/src/views/Calendar.vue#L279
for an example how to properly replace a variable in a translated string.

Reply:
@raimund-schluessler Thank you!